modules/ROOT/pages/cloudhub-impaired-worker.adoc

```diff
@@ -3,8 +3,11 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-CloudHub performs constant health checks on the underlying virtual machines where Mule applications are running. When an instance is impaired, there is hardware or software failure that may negatively impact the running application deployed on the instance. Examples of hardware and software failures include network disconnection, memory issues, server hardware failures, file corruption, etc.
+CloudHub performs constant health checks on the underlying virtual machines where Mule applications are running. A worker is impaired when there is hardware or software failure that negatively impacts the running application deployed on the worker. Examples of hardware and software failures include network disconnection, memory issues, server hardware failures, file corruption, and so on.
```

Comment:
Missing a word: should be "when there is a hardware or software failure..."
Also "network disconnection" sounds overly specific. How about "network unavailability" which could encompass outages, performance issues, and more.